tests/crash_similarity_support.py

```diff
@@ -0,0 +1,5 @@
+func = ['js::jit::MakeMRegExpHoistable ', ' AppKit@0x7be82f ', ' __RtlUserThreadStart ', ' xul.dll@0x1ade7cf ', 'XUL@0x7bd20f', 'libxul.so@0xe477b4 ']
+
+stack_trace = "js::GCMarker::processMarkStackTop | js::GCMarker::drainMarkStack | js::gc::GCRuntime::incrementalCollectSlice | js::gc::GCRuntime::gcCycle | js::gc::GCRuntime::collect | JS::StartIncrementalGC | nsJSContext::GarbageCollectNow | nsTimerImpl::Fire | nsTimerEvent::Run | nsThread::ProcessNextEvent | NS_ProcessPendingEvents | nsBaseAppShell::NativeEventCallback | nsAppShell::ProcessGeckoEvents | CoreFoundation@0xa74b0 | CoreFoundation@0x8861c | CoreFoundation@0x87b15 | CoreFoundation@0x87513 | HIToolbox@0x312ab | HIToolbox@0x310e0 | HIToolbox@0x30f15 | AppKit@0x476cc | AppKit@0x7be82f | CoreFoundation@0x9e3a1 | AppKit@0xc56609 | AppKit@0xc9e7f7 | AppKit@0xc9e387 | AppKit@0xc567a9 | AppKit@0xc5867b | AppKit@0xc57ccc | AppKit@0xc5a9c2 | AppKit@0x47c2ed | AppKit@0x47c304 | AppKit@0xcdcf03 | AppKit@0xc56e2b | AppKit@0xc579af | AppKit@0xcdcee2 | AppKit@0xc5e77b | AppKit@0xc9897a | AppKit@0xc9c88c | AppKit@0xc7f10e"
+
+top_ten_stack_traces = ['js::gcmarker::processmarkstacktop', 'js::gcmarker::drainmarkstack', 'js::gc::gcruntime::incrementalcollectslice', 'js::gc::gcruntime::gccycle', 'js::gc::gcruntime::collect', 'js::startincrementalgc', 'nsjscontext::garbagecollectnow', 'nstimerimpl::fire', 'nstimerevent::run', 'nsthread::processnextevent']
```

Comment:
They are the frames of a stack trace, starting from the frame where the crash happened to the frame that is 10 steps before. So you can call them top or bottom depending on how you show the stack trace.

Let's say you have the following stack trace, where "->" means "calls" and the crash is happening in function L:
A -> B -> C -> D -> E -> F -> G -> H -> I -> L

If you depict the stack trace this way, then what we're considering are the bottom 10:
A
B
C
D
E
F
G
H
I
L

If you depict the stack trace this way (like Socorro does, e.g. https://crash-stats.mozilla.com/report/index/79005970-51a2-41e7-96b2-565f42170308#tab-details), then what we're considering are the top 10:
L
I
H
G
F
E
D
C
B
A